site/public/js/forum.js

```diff
@@ -1895,6 +1895,52 @@ function bookmarkThread(thread_id, type) {
     });
 }
 
+// eslint-disable-next-line no-unused-vars
+function markThreadUnread(thread_id) {
+    // eslint-disable-next-line no-undef
+    const url = `${buildCourseUrl(['forum', 'threads', 'unread'])}`;
```

Comment:
I think you can add buildCourseUrl to the list of global functions at the top of the file and then you can remove the es lint disable for undefined